### DISCUSSION TITLE
How long does it take?

### DISCUSSION THREAD

**Discussion by MlooneyHV** [grahampugh/erase-install#219] [Q&A]
How long does it take for prepare the macOS for installation? One of the machines i've been testing it on has been in this state for over 45 minutes.

EDIT: The process was hanging because it was trying to use an already downloaded macOS Monterey installer. Once deleted, it downloaded it's own and the process completed successfully.

**Comment by grahampugh** [comment]
If you're testing, try using `--no-fs` so you can see the output of your erase-install.sh script (if you're running it locally).
Also if you use `--depnotify` you get a progress bar.
